doc/ref_cert/RC2/chapters/chapter04.md

```diff
@@ -3,3 +3,38 @@
 # 4. CNF Test Cases and Requirements Traceability
 
 ## Table of Contents
+* [4.1 Introduction](#4.1)
+* [4.2 Selection Criteria](#4.2)
+* [4.3 Traceability Matrix](#4.3)
+  * [4.3.1 Test Case Traceability to CNF Interoperability Requirements](#4.3.1)
+  
+<a name="4.1"></a>
+## 4.1 Introduction
+The scope of this chapter is to identify and list down test cases based on requirements defined in [Reference Architecture-2 (RA-2)](../../../ref_arch/kubernetes/README.md). This will serve as traceability between test cases and requirements for CNF interoperability.
```

Comment:
requirements relevant to CNF use of infrastructure

Reply:
from #1496 "Clear position agreed between CNTT and CNCF: RA2 shoudl only deal with requirements that workloads have in order to interoperate with an RA2-compliant platform. Cloud native best practices and the requirements relating to those will be handled in either the CNCF TUG or CNF Working Group."

Reply:
See my comment above https://github.com/cntt-n/CNTT/pull/2200#discussion_r563931220

Reply:
In the meeting on January 28, 2021, it was decided that the title "[4.3.1 Test Case Traceability to CNF Interoperability Requirements](https://github.com/cntt-n/CNTT/blob/Chapter-4-framework/doc/ref_cert/RC2/chapters/chapter04.md#43-traceability-matrix)" to "4.3.1 Test Case Traceability to RA2 Requirements"